src/cdk-experimental/menu/context-menu.spec.ts

```diff
@@ -399,7 +399,8 @@ describe('CdkContextMenuTrigger', () => {
       };
     }
 
-    it('should throw an error if context and menubar trigger share a menu', () => {
+    // TODO: seems to work now, should we go out of our way to break it?
```

Comment:
@jelbourn sharing menus seems to work fine with the new setup, should I allow it or add some code to catch it and throw? It seems like it could be useful in some cases, e.g. if your app has the same options in the "Edit" menu and context menu

Reply:
I don't see any reason to prevent it